src/parquet/column/column-reader-test.cc

```diff
@@ -45,42 +45,6 @@ namespace test {
 
 class TestPrimitiveReader : public ::testing::Test {
  public:
-  void MakePages(const ColumnDescriptor *d, int num_pages, int levels_per_page) {
```

Comment:
Definitely good to factor this code out of here and `scanner-test` for better reuse